### DISCUSSION TITLE
Hosting Discord Bots?

### DISCUSSION THREAD

**Discussion by Vlad-Galeev** [deta/cloud-docs#421] [Help] [answered]
I tried putting a simple bot on Deta, and it doesn't respond to slash commands on Discord. Is there anything I might be missing?

**Comment by jnsougata** [comment, marked as answer]
Interaction only bot? Cause micro doesn't support websockets!

**Reply by Vlad-Galeev** [reply]
What do you mean by interaction only? I am using the py-cord library.

**Reply by jnsougata** [reply]
well in that case it won't work. micros don't support websockets as I said earlier

**Reply by Vlad-Galeev** [reply]
Thank you for your help.
